showcase/src/app/demo/header/header.component.ts

```diff
@@ -47,7 +47,7 @@ export class HeaderComponent implements OnInit {
 									<mat-icon starkSvgViewBox svgIcon="dots-vertical"></mat-icon>
 							</button>
 							<button color="alert" mat-mini-fab>
```

Comment:
Could you please remove this `<button color="alert" mat-mini-fab>` ?
Your `stark-app-logout` component contains already a button...
For info, the HTML code of stark-logout:

```html
<button mat-mini-fab color="alert" (click)="logout()" 
	[matTooltip]="'STARK.APP_LOGOUT.TITLE' | translate">
	<mat-icon starkSvgViewBox [svgIcon]="icon"></mat-icon>
</button>
```